virttest/qemu_devices/qcontainer.py

```diff
@@ -2180,11 +2180,10 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
         """
 
         devices = []
-        if bus_extra_params:
-            bus_params = (dict([_.split('=')
-                                for _ in bus_extra_params.split(',') if _]))
         # For virtio devices, generate controller and create the port device
         if serial_type.startswith('virt'):
+            bus_params = (dict([_.split('=')
+                                for _ in bus_extra_params.split(',') if _]))
```

Comment:
The newly introduced param should work not only for virtio serial controllers, although our tests may not have such requirements currently.

Reply:
@luckyh updated, thanks!